### PR TITLE
[WIP] Started adding rexray config.

### PR DESCRIPTION
**Pull request by jeid64**
Someone still needs to do the Python test setting where the options.json gets set to use the new rexray options.

### CODE REVIEW

**Comment by benclarkwood** [comment]
@jeid64 be the change you wish to see in the world.

Looks good.

File: universe/marathon.json.mustache [hunk 1]
```diff
@@ -50,15 +50,26 @@
...
           {{/storage.host-volume}}
-          {{^storage.host-volume}}
+          {{^storage.external-persistent-volume-name}}
             {
               "persistent": {
-                "size": {{storage.persistent-volume-size}}
+                "size": {{storage.local-persistent-volume-size}}
               },
               "mode": "RW",
               "containerPath": "jenkins_home"
```
Comment (benclarkwood) [inline universe/marathon.json.mustache]: Wait. Is this the right container path? Host volume is at a different place?

Comment (jeid64) [reply universe/marathon.json.mustache]: External/local persistent volumes have to be relative to the sandbox, so there's a JENKINS_HOME variable that changes it to be sandbox/jenkins_home

File: universe/config.json [hunk 2]
```diff
@@ -42,14 +42,23 @@
                     "description": "The location of a volume on the host to be used for persisting Jenkins configuration and build data. The final location will be derived from this value plus the name set in `name` (e.g. `/mnt/host_volume/jenkins`). Note that this path must be the same on all DC/OS agents.",
                     "type": "string"
                 },
-                "persistent-volume-size": {
+                "local-persistent-volume-size": {
```
Comment (benclarkwood) [inline universe/config.json]: This is a breaking "API" change.

Leave it as is. I don't see a reason to have a separate field for the two.

I would actually make external a boolean flag, and then consume the name if the boolean is true.

File: tests/scale/test_load.py [hunk 3]
```diff
@@ -46,7 +46,8 @@ def test_scaling_load(master_count,
                       run_delay,
                       cpu_quota,
                       work_duration,
-                      mom):
+                      mom,
+                      external_volume):
```
Comment (colin-msphere) [inline tests/scale/test_load.py]: I think this may be passed through as a string (`"True"`). This could cause issues on some `if` checks.

**Pull request merged**